doctr/datasets/cord.py

```diff
@@ -54,6 +54,8 @@ def __init__(
         self.data: List[Tuple[str, Dict[str, Any]]] = []
         self.train = train
         self.sample_transforms = sample_transforms
+        np_dtype = np.float32
```

Comment:
On the constructor call line 50, we need to specify the value for `convert_to_relative`